packages/wagtail/modules/services/Wagtail/PagePreviewAPI/index.ts

```
@@ -0,0 +1,15 @@
+import { WagtailAPI } from '..'
+import { IPage } from '../PagesAPI/types'
+
+export class PagePreviewAPI extends WagtailAPI {
+  static namespace = `base/page_preview`
+
+  static get(token: string, contentType: string): Promise<IPage> {
+    return this.axios.get(`${this.baseURL}/${this.namespace}/`, {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)